src/style.css

```diff
@@ -0,0 +1,101 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.flex {
+  height: 300px;
+  min-width: 300px;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  text-align: center;
+  line-height: 300px;
+  flex-grow: 1;
+  flex-basis: 33%;
+}
+
+.box_1 {
+  background-color: hsl(0,100%,50%);
+  order: 1;
+}
+
+.box_2 {
+  background-color: hsl(0,100%,40%);
+  order: 2;
```

Comment:
You don't need to provide order property to those elements, which order should be the same as in HTML.